ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -535,4 +534,4 @@
   ;; Timeout used because toggling key for route-changes table directly may cause delay in rendering the content with large data set.
   ;; Thus disabling of UI components must happen before table model change because otherwise table rendering delays those as well.
   (.setTimeout js/window #(e! (->InitiateRouteModelUpdate)) 0)
-  (update app :all-route-changes-checkbox not))
+  app)
```

Comment:
In this form `ToggleShowNoChangeRoutes` does not contain any functionality, which makes it is useless and should be removed.
As the comments state, event was implemented to separate rendering the checkbox and rendering the state model changes into different js event loop events, because latter one blocks UI in certain cases with large data for some time.

Whichever approach is taken, please check that there's no bad UX using different browsers.

Reply:
So I did break up the entire fix that you made earlier. I did test this only with services that had small amount of routes. I'll try change it back. Thanx